src/neo-vm/Types/Buffer.cs

```diff
@@ -19,7 +19,7 @@ namespace Neo.VM.Types
     /// <summary>
     /// Represents a memory block that can be used for reading and writing in the VM.
     /// </summary>
-    [DebuggerDisplay("Type={GetType().Name}, Value={System.BitConverter.ToString(InnerBuffer).Replace(\"-\", string.Empty)}")]
+    [DebuggerDisplay("Type={GetType().Name}, Value={System.BitConverter.ToString(GetSpan().ToArray()).Replace(\"-\", string.Empty)}")]
```

Comment:
Can we use Convert.ToHexString here? I realize it's for debugger display, so optimizing isn't that important, but ToHexString avoids converting the span to an array and replacing the dashes

Reply:
Done